test/integration/targets/vmware_content_deploy_template/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+# Test code for the deploy VM from content library template.
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_datacenter: true
+
+- when: vcsim is not defined
+  block: 
+    - name: Deploy VM from content library template
```

Comment:
Run this task again to check idempotency of module object.

Reply:
Tested with all parameters

Reply:
I agree with @Akasurde , we want to run this task again to see that we hit `msg="Virtual Machine '%s' already Exists.` and `changed=False`.

Reply:
Thanks @jillr , I misread Abhijeet's comment. Modified the test to rerun the task for idempotency